### ISSUE TITLE
How to install dependencies in dev-requires?

### ISSUE THREAD

**Issue by Mariatta** [takluyver/flit#184]
I have the following in my `pyproject.toml`
```
[tool.flit.metadata]
....
requires = ["uritemplate>=3.0.0"]
dev-requires = ["pytest>=3.0.0",
    "pytest-asyncio",
    "pytest-tornasync",
    "pytest-runner>=2.11.0",
    "sphinx",
    "aiohttp",
    "treq",
    "twisted[tls]",
    "tornado",
]
...
```

Then I tried to install this using `flit install --deps develop`.
It would only install `uritemplate`, and nothing inside `dev-requires`.

Maybe I misunderstood what `--deps` is for.

How can I install the dependencies inside `dev-requires`?

Thanks.

Edit: using Flit 1.0

**Comment by takluyver** [comment]
No, that's meant to work; I think it got lost in PR #170.

PR #181 will also touch this machinery.

**Comment by Mariatta** [comment]
Closing this issue since it has been fixed in #181.
